scripts/deploy/deployUniversalVerifier.ts

```diff
@@ -14,7 +14,7 @@ import { contractsInfo } from "../../helpers/constants";
 
 async function main() {
   const config = getConfig();
-  const chainId = hre.network.config.chainId;
+  const chainId = parseInt(await hre.network.provider.send("eth_chainId"), 16);
```

Comment:
Create some function `getChainId()` in `helperUtils.ts` like this
```
export async function getChainId() {
  return parseInt(await hre.network.provider.send("eth_chainId"), 16);
}
```
and replace all places where we get `hre.network.config.chainId` or `hre.network.provider.send("eth_chainId"), 16)` with this `getChainId()`